docs/Tutorials/Quickstarts/Private-Network-Quickstart.md

```diff
@@ -436,14 +417,27 @@ Compiling truffle/DeployedAddresses.sol...
   3 passing (37s)
 ```
 
-Continue with the Truffle tutorial steps in the [Creating a user interface to interact with the smart contract](https://truffleframework.com/tutorials/pet-shop#creating-a-user-interface-to-interact-with-the-smart-contract) section.
 
-We've already connected the private network to MetaMask, so you can skip the [Installing and configuring MetaMask](https://truffleframework.com/tutorials/pet-shop#installing-and-configuring-metamask) section.
+Once these tests are successful, it builds a container for the Pet Shop DApp and deployes it, binding it to port 3001 on your machine. 
+
+Open a new tab in browser and navigate to `http://localhost:3001` which should have the Pet Shop running, and you can adopt 
```

Comment:
Specify that the browser you use has to be the one with Metamask enabled and account loaded.

Reply:
:+1: